src/main/java/edu/kis/powp/jobs2d/command/CommandImporter.java

```diff
@@ -0,0 +1,9 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.io.IOException;
+
+public interface CommandImporter   {
+
+	CompoundCommand importCommand(String filepath) throws IOException;
+	void saveCommand(String filepath, CompoundCommand command) throws IOException;
```

Comment:
Tangling with files is unnecessary. This is no OCP and SRP approach. Instead of filepath I expect a command string. Such an approach would allow e.g., an easy shift from files to a textfield.